library/build.gradle

```diff
@@ -20,7 +20,7 @@ dependencies {
     compile project(':third_party:disklrucache')
     compile project(':annotation')
     compile "com.android.support:support-v4:${SUPPORT_V4_VERSION}"
-
+    compile "com.android.support:appcompat-v7:${SUPPORT_V7_VERSION}"
```

Comment:
I think a better way would be to use a provided scope and `try { loadVectorWithV7 } catch(ClassNotFoundException ex) { loadVectorWithV7 }` (could cache failure so the weight of the exception only happens once). This way there's no need for proguard.